### ISSUE TITLE
Is this module still required after cordova-ios v5.0.0

### ISSUE THREAD

**Issue by masimplo** [akofman/cordova-plugin-add-swift-support#62]
In cordova-ios@5.0.0 this PR has been merged https://github.com/apache/cordova-discuss/pull/107. 
I am not sure if this plugins is required (or conflicts) with v5.0.0 of cordova-ios. I am currently having some build issues with swift version warnings after upgrading to 5.0.0 and wander if this is the culpit.

**Comment by akofman** [comment]
Indeed, I didn't test it but I heard the last Cordova version is now supporting Swift. So yes you should be able to remove this plugin.

**Comment by bbjay** [comment]
just tried to go without this plugin and discovered that cordova does not alter the SWIFT_OPTIMIZATION_LEVEL build setting to -0none. So debugging does not work with plain cordova, and this plugin might still be valuable.
